Code/skyrim_ui/src/app/components/server-list/server-list.component.ts

```diff
@@ -138,6 +141,11 @@ export class ServerListComponent {
       favoriteServers[`${ favoriteServer.ip }:${ favoriteServer.port }`] = favoriteServer;
     }
     this.favoriteServers.next(favoriteServers);
+
+    settingService.fontSizeChange.subscribe(fontSize => {
```

Comment:
manage subscription `.pipe(takeUntil(this.destroy$))`

Reply:
I've switched it to using an observable instead now. This is used by an [AsyncPipe ](https://angular.io/api/common/AsyncPipe#description) in the template, which handles unsubbing when the component is destroyed.